runtime/criusupport/criusupport.cpp

```diff
@@ -321,18 +360,18 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 	const char *nlsMsgFormat = NULL;
 	UDATA msgCharLength = 0;
 	IDATA systemReturnCode = 0;
-	criuSetUnprivilegedFunctionPointerType criuSetUnprivileged = NULL;
 	const char *dlerrorReturnString = NULL;
+	bool setupCRIU = false;
```

Comment:
Should this default to true to handle the case where you do not call the setup function? Either that or set it to true if the field value is not NULL below.